sql/core/src/main/scala/org/apache/spark/sql/sources/interfaces.scala

```diff
@@ -151,6 +151,7 @@ trait StreamSourceProvider {
 @InterfaceStability.Unstable
 trait StreamSinkProvider {
   def createSink(
+      schema: StructType,
```

Comment:
The interface can't just be changed like this. This breaks backwards compatibility ...